build.sbt

```diff
@@ -22,7 +22,8 @@ initialize := {
   initialize.value
 }
 
-// akka.AkkaBuild.buildSettings
```

Comment:
Do we know why that `akka.AkkaBuild.buildSettings` was commented out, or why it was needed to begin with?

it was removed in https://github.com/akka/akka/commit/be7790c7b975b59a58f93b01a76cae744f761cb3

Reply:
I think it was to not give the root project all cross versions, to be able to limit building typed only on 2.12+ (because of this: https://github.com/sbt/sbt/issues/3465 )